tests/sanity_helpers.py

```diff
@@ -36,13 +39,14 @@ def health_check(self):
         )
         self.ceph_cluster.cluster_health_check(timeout=60)
 
-    def create_resources(self, pvc_factory, pod_factory, run_io=True):
+    def create_resources(self, pvc_factory, pod_factory, dc_pod_factory, run_io=True):
```

Comment:
I don't think we need to use pod (not DC) anymore

Reply:
will replace pod_factory with dc_pod_factory in next commit